llama_stack/providers/tests/safety/conftest.py

```diff
@@ -63,6 +71,18 @@ def pytest_addoption(parser):
         default=None,
         help="Specify the safety model to use for testing",
     )
+    parser.addoption(
```

Comment:
hm, this feels like it won't scale to all kinds of providers. can we use environment variables somehow?

Reply:
done.